tests/RobotFramework/tests/tedge/tedge_cert_create.robot

```diff
@@ -78,54 +78,114 @@ Run tedge cert create with cloud profile
     Should Match Regexp    ${subject}    pattern=^subject=CN = ${THIRD_DEVICE_SN},.+$
 
 Device ID derivation
+    Execute Command    tedge config get device.id    exp_exit_code=1
+    Execute Command    tedge config set c8y.device.cert_path /etc/tedge/device-certs/tedge-certificate@default.pem
+    Execute Command    tedge config set c8y.device.key_path /etc/tedge/device-certs/tedge-private-key@default.pem
+    Execute Command
+    ...    tedge config set c8y.device.cert_path --profile foo /etc/tedge/device-certs/tedge-certificate@foo.pem
+    Execute Command
+    ...    tedge config set c8y.device.key_path --profile foo /etc/tedge/device-certs/tedge-private-key@foo.pem
+
+    ### No device ids are explicitly set
+    # create certificate without explicit device.id in tedge config setting
```

Comment:
I wouldn't call this "explicit" as inferring a value from configuration is really be explicit.

```suggestion
    # create certificate without device.id in tedge config setting
```

Reply:
rephrased some comment lines in ef49eeb23438484e78e27beffeba18a9e9e950b5